src/controllers/update/update.manage.trusts.tell.us.about.the.legal.entity.controller.ts

```diff
@@ -30,11 +30,17 @@ const getPageProperties = (trust, formData, errors?: FormattedValidationErrors)
       title: 'Tell us about the legal entity',
     },
     pageData: {
-      trustData: { trustName: trust?.trust_name },
+      trustData: {
+        trustName: trust?.trust_name,
+        creationDateDay: trust?.creation_date_day,
```

Comment:
See my new PR changes, I've had to remove the use of creationDate fields for validation and instead lookup the trust in the validation rule.

Reply:
Updated to include these changes.

Reply:
Looks to be addressed. Resolving.